jaxrs-server/src/main/java/io/micronaut/jaxrs/runtime/ext/bind/JaxRsHttpHeaders.java

```diff
@@ -58,7 +58,12 @@ public List<String> getRequestHeader(String name) {
 
     @Override
     public String getHeaderString(String name) {
-        return String.join(",", httpHeaders.getAll(name));
+        List<String> all = httpHeaders.getAll(name);
+        if (all.isEmpty()) {
```

Comment:
should we use `CollectionUtils::isEmpty` to protect against NPE?

Reply:
Good call...  There's no contract that it can't be null, so you're right 👍